dist2src/cli.py

```diff
@@ -491,6 +545,7 @@ def convert_with_prep(ctx, origin, dest):
     # configure packit
     ctx.invoke(add_packit_config, dest=Path(dest_dir))
     ctx.invoke(copy_spec, origin=origin_dir, dest=dest_dir)
+    ctx.invoke(copy_all_sources, origin=origin_dir, dest=dest_dir)
```

Comment:
This copying from `SOURCES` to `SPECS` is very weird and scares me every time (until I recall that there was #11).

Could this maybe made a separate commit, so that it's not hidden under "Add spec-file for the distribution"?

Reply:
I was lazy to separate the change, can do

Reply:
/o\ now I understand what you're asking, read it wrong the first time

Reply:
Yes, this was not a "fix your commits in the PR" comment but a "fix how this code makes commits" comment :D